apps/web-giddh/src/app/ledger/components/updateLedgerEntryPanel/updateLedgerEntryPanel.component.ts

```diff
@@ -1735,7 +1734,7 @@ export class UpdateLedgerEntryPanelComponent implements OnInit, AfterViewInit, O
                 tcsTotal: 0,
                 tdsTotal: 0,
                 balanceDue: this.vm.selectedLedger.total.amount,
-                grandTotal: this.vm.selectedLedger.total.amount,
+                grandTotal: this.vm.selectedLedger.entryVoucherTotals.amountForAccount,
```

Comment:
Since entryVoucherTotals is optional value, please use 
```suggestion
                grandTotal: this.vm.selectedLedger.entryVoucherTotals?.amountForAccount,
```